InventoryManagementSystem/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿// See https://aka.ms/new-console-template for more information
+using InventoryManagementSystem.Domain;
+using InventoryManagementSystem.Domain.Reader;
+using System.Reflection;
+using System.Reflection.Emit;
+
+IReader cr = new ConsoleReader();
+Inventory inventory = new Inventory(cr);
+
+Console.WriteLine("Welcome to the Product Inventory Management System");
+while (true)
+{
+    var options =
```

Comment:
we can have the **options** variable outside the while loop, no need to instantiate it every loop